packages/core/src/Toggle/Toggle.stories.tsx

```diff
@@ -1,30 +1,276 @@
+
+    return (
+      <StoryStage display='flex'>
+        <Toggle isOn={isOn} onToggle={onToggle} />
+        <Toggle isOn={isOn} onToggle={onToggle} height={16} />
+      </StoryStage>
+    )
+  },
+  argTypes: {
+    isOn: {
+      type: 'boolean',
+    },
+    disabled: {
+      type: 'boolean',
+      defaultValue: false,
+    },
+  },
+}
+
+export const WithIcon: ToggleStory = {
+  render: () => {
+    const [isOn, setIsOn] = useState(false)
+    const onToggle = () => setIsOn(!isOn)
+    const Icon = isOn ? CheckIcon : CloseIcon
+
+    return <Toggle isOn={isOn} onToggle={onToggle} icon={<Icon p={1} />} />
+  },
+  argTypes: {
+    isOn: {
+      type: 'boolean',
+    },
+    disabled: {
+      type: 'boolean',
+      defaultValue: false,
+    },
+  },
+}
+
+const meta: Meta<typeof Toggle> = {
+  title: 'Actions/Toggle',
+  component: Toggle,
+  parameters: {
+    docs: {
+      page: () => (
+        <ThemeProvider>
+          <Hero name='Toggle' img={heroImage}>
+            Buttons allow users to trigger an action or event with a single click. For example, you can use a
+            button for allowing the functionality of submitting a form, opening a dialog, canceling an action,
+            or performing a delete operation.
+          </Hero>
+
+          <TableOfContents
+            links={['Overview', 'Props', 'States', 'Examples', 'Usage', 'Related Components']}
+          />
+
+          <Section heading='Overview'>
+            <Text textStyle='paragraph'>
+              A toggle allows users to independently control binary options (a single state that is either on
+              or off), such as a setting. When a toggle is interacted with, it should take immediately effect
+              and it's appearance should clearly indicate when it's activated or deactivated state.
+            </Text>
+          </Section>
+
+          <Section heading='Props'>
+            <Primary />
+            <ArgsTable story={PRIMARY_STORY} />
+          </Section>
+
+          <Section heading='States'>
+            <Flex my={4} borderRadius='xl' bg='primary.light'>
+              <Image src={statesImage} />
+            </Flex>
+          </Section>
+
+          <Section heading='Examples'>
+            <StoryHeading storyName='Sizes' storyTitle={meta.title} />
+            <Text textStyle='paragraph'>
+              Toggles support two toggle sizes that consists of a base size, and a small size. These sizes
+              allow for hierarchy, emphasis, larger/smaller touch targets, and flexibility.
+            </Text>
+            <LiveDemo code={Sizes} />
+
+            <StoryHeading storyName='With Icon' storyTitle={meta.title} />
+            <Text textStyle='paragraph'>
+              Toggles support two toggle sizes that consists of a base size, and a small size. These sizes
+              allow for hierarchy, emphasis, larger/smaller touch targets, and flexibility.
+            </Text>
+            <LiveDemo code={WithIcon} />
```

Comment:
It looks like there's an error when viewing this part of the docs in Storybook 
![CleanShot 2023-07-11 at 09 23 10](https://github.com/priceline/design-system/assets/3260642/31b285ba-594c-46e5-ba83-c87c39d76bf4)

Reply:
This is due to the LiveDemo component, which has been removed